security.md

```diff
@@ -16,6 +16,25 @@ responded.
 To report a possible security vulnerability, please email `security@spark.apache.org`. This is a
 non-public list that will reach the Apache Security team, as well as the Spark PMC.
 
+<h2>Frequently Asked Questions</h2>
+
+<h3>During a security analysis of Apache Spark, I noticed that Spark allows for remote code execution, is this an issue?</h3> 
+
+As with other similar projects like
+[Apache Flink](https://flink.apache.org/what-is-flink/security/#during-a-security-analysis-of-flink-i-noticed-that-flink-allows-for-remote-code-execution-is-this-an-issue), 
```

Comment:
replace "As with other similar projects like
[Apache Flink](https://flink.apache.org/what-is-flink/security/#during-a-security-analysis-of-flink-i-noticed-that-flink-allows-for-remote-code-execution-is-this-an-issue)," with "No, this is not considered an issue or a vulnerability **in itself**, because remote code execution is fundamental to Apache Spark's design and purpose."